components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1618,10 +1619,11 @@ void AdsServiceImpl::StartRemoveOnboardingTimer() {
       base::TimeDelta::FromSeconds(timer_offset_in_seconds),
           base::BindOnce(&AdsServiceImpl::RemoveOnboarding, AsWeakPtr()));
 
-  auto time = base::TimeFormatFriendlyDateAndTime(
-      base::Time::FromDoubleT(timestamp_in_seconds));
+  const std::string friendly_date_and_time =
+      base::UTF16ToUTF8(base::TimeFormatFriendlyDateAndTime(
+          base::Time::FromDoubleT(now_in_seconds + timer_offset_in_seconds)));
```

Comment:
Isn't `now_in_seconds + timer_offset_in_seconds` equal to `timestamp_in_seconds`?

Reply:
In this situation, it is not as if the timestamp has expired we trigger the timer after 1 minute